### PR TITLE
Add discord bot which integrates with the existing NER model

### PR DESCRIPTION
**Pull request by pncnmnp**
The bot contains logging identification and basic name tagging functionality

### CODE REVIEW

**Comment by CallmeMehdi** [comment]
Everything looks good @pncnmnp it works perfectly. But can you please change the branch to dev? We need to merge discord into dev.

**Comment by pncnmnp** [comment]
Changed base to dev

File: src/discord_bot.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+async def on_message(message):
+    if message.author == client.user:
+        return
+
+    print(message.content)
+    if message.content.startswith('$tag'):
+        content = message.content.replace("$tag", "")
+        prediction = guild_obj.bento_service.predict([{"sentence": content}])
+
+        tokens = prediction[0].split(",")
```
Comment (pncnmnp) [inline src/discord_bot.py:50]: They are not integrated!

**Comment by pncnmnp** [comment]
**This pull request should not be merged. The integration part with NER is incomplete.**
I have not deployed the tokens in line 50 of discord_bot.py.

**Review by pncnmnp — COMMENTED** [review]
I have fixed the changes. @CallmeMehdi can you review them.

File: src/discord_bot.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import discord
+import bentoml
+import itertools
+import os
+
+# Create a DISCORD_TOKEN (https://discordpy.readthedocs.io/en/latest/discord.html#discord-intro)
+# Store in a .env file with the following structure
+"""
+# .env
+DISCORD_TOKEN={your-discord-token}
```
Comment (CallmeMehdi) [inline src/discord_bot.py:12]: This will be then changed in the deployment right?

**Review by CallmeMehdi — APPROVED** [review]
Everything seems great!

**Comment by CallmeMehdi** [comment]
We don't need to delete the discord branch as we will work on it in the future.

File: src/discord_bot.py (hunk 2 above)
Comment (pncnmnp) [reply src/discord_bot.py:12]: Yes we need to create a `.env` file and add the tokens there without making a commit of `.env`.

**Pull request merged**